### PR TITLE
Data accessors for victory-line

### PR DESCRIPTION
**Pull request by dandelany**
Adds data accessors to VictoryLine - see FormidableLabs/victory#84

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "victory-line",
-  "version": "0.7.0",
+  "version": "0.8.0",
```
Comment (boygirl) [inline package.json]: NIT: no manual version bumps. We've got a script for that https://github.com/FormidableLabs/builder-victory-component/blob/master/dev/DEVELOPMENT.md#releases

**Comment by boygirl** [comment]
cc/ @dandelany looks great! Consider it approved once your supporting work in `victory-util` gets released

File: demo/demo.jsx [hunk 2]
```diff
@@ -1,7 +1,8 @@
 /*global window:false */
 import React from "react";
-import {VictoryLine} from "../src/index";
 import _ from "lodash";
+import {VictoryLine} from "../src/index";
+import {VictoryChart} from "victory-chart";
```
Comment (boygirl) [inline demo/demo.jsx]: please add `victory-chart` as a dev dependency

**Comment by dandelany** [comment]
Thanks - added 'victory-chart' to devDeps.

**Pull request merged**